libs/sdk-ui-kit/src/Header/generateHeaderMenuItemsGroups.ts

```diff
@@ -41,6 +46,7 @@ export function generateHeaderMenuItemsGroups(
 
     const shouldHidePixelPerfectExperience = shouldHidePPExperience(featureFlags);
     const isFreemiumCustomer = isFreemiumEdition(featureFlags.platformEdition.toString());
+    const workspaceRef = enableRenamingProjectToWorkspace === true ? "workspace" : "project";
```

Comment:
`=== true` is not needed, is it?

Reply:
I have tested with test cases and it's redundant thanks I have changed it